### ISSUE TITLE
Feature: nudge loop points

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#46]
As per Bob Elling, it's nice to have a way to nudge the loop points in a couple of ways:
1) nudge left loop point left/right
2) same for right loop point
3) same for both loop points at the same time

It's also nice to have a modifier key to make the nudge bigger.

**Comment by danlyke** [comment]
Proposal: Maybe we need to rename the Lyrics tab, but let's just use the controls in the lyrics tab to control the loop points. This'll get us loop adjustment almost immediately, I'll be happier at being able to tune loop points, and we can work forwards from there.

And, yes, I have a patch pretty much ready to go.

**Comment by mpogue2** [comment]
OK, we've been using Dan's fields for loop start/endpoint setting for a while (thanks, Dan!).  To enhance this, I've replaced the lineEdit fields with dateTimeEdit fields, which allows for nudging with arrow keys or clicking on the up/down buttons, and for editing (by section) using 0-9 keys or DEL.  They also prevent setting times that are not within the song. I also added a Test Loop button, which is shown only for Patter.  The menu item "Music > Test Loop" is always available.

Only thing that's not in is disallowing making the end point earlier than the start point (and vise versa).  I'll open a separate Issue for that.

Branch: "Nudge".  This is a fairly low-risk change, so if it looks OK by later tonight, I'll commit to Master.  Commit: a2b20c3b024fcc47f8014a1f47b028dc452e7649 .
